packages/strapi-helper-plugin/lib/src/components/Select/index.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import ReactSelect, { components } from 'react-select';
+import PropTypes from 'prop-types';
+import { Carret } from '@buffetjs/icons';
+import { useTheme } from 'styled-components';
+
+const DropdownIndicator = props => {
+  const theme = useTheme();
+
+  return (
+    <components.DropdownIndicator {...props}>
+      <Carret fill={theme.main.colors.grey} />
+    </components.DropdownIndicator>
+  );
+};
+
+export const Select = ({ children, onChange, selectedValue, ...props }) => {
+  const theme = useTheme();
+  const childrenArray = React.Children.toArray(children);
+
+  const options = childrenArray.map(child => ({
+    value: child.props.value,
+    label: child.props.children,
+  }));
+
+  const selectedOption = options.find(({ value }) => value === selectedValue);
+
+  return (
+    <ReactSelect
+      options={options}
+      onChange={({ value }) => onChange(value)}
+      components={{ DropdownIndicator }}
+      styles={{
+        indicatorsContainer: provided => ({
+          ...provided,
+          background: theme.main.colors.brightGrey,
+        }),
+        dropdownIndicator: provided => ({
+          ...provided,
+          padding: '0 1.8rem',
+          margin: 0,
+        }),
+        indicatorSeparator: provided => ({
+          ...provided,
+          display: 'none',
+        }),
+        control: provided => ({
+          ...provided,
+          border: `1px solid ${theme.main.colors.border}`,
+          cursor: 'pointer',
+        }),
+      }}
+      value={selectedOption}
+      {...props}
+    />
+  );
+};
+
+export const Option = () => <></>;
```

Comment:
I don't understand this component, can you explain it a bit for me?

Reply:
This is the beauty! This one does nothing: it's here to replicate the API of the HTML select element. And it's useful only because the Select component iterates on its "value" prop :blush:

Reply:
Can you put a comment on it, so that it will not be deleted, please? 😅